.devcontainer/cpp/devcontainer-metadata-vscode.json

```diff
@@ -12,7 +12,7 @@
         "ms-vscode.cmake-tools@1.18.42",
         "ms-vscode.cpptools@1.20.5",
         "ms-vsliveshare.vsliveshare@1.0.5932",
-        "SonarSource.sonarlint-vscode@4.7.0",
+        "sonarsource.sonarlint-vscode@4.7.0",
```

Comment:
In the other files it mentions v4.6.0 instead of v4.7.0. Should those versions be in sync?

Reply:
Yes they should be in-sync. Will update.